src/kudu/consensus/raft_consensus.h

```diff
@@ -961,6 +967,7 @@ class RaftConsensus : public std::enable_shared_from_this<RaftConsensus>,
   scoped_refptr<Counter> follower_memory_pressure_rejections_;
   scoped_refptr<AtomicGauge<int64_t>> term_metric_;
   scoped_refptr<AtomicGauge<int64_t>> num_failed_elections_metric_;
+  bool new_leader_detected_failsafe_;
```

Comment:
mind adding a doc comment for this variable? it's not immediately clear what it's supposed to do or in what was it is a "failsafe"